brainscore_language/benchmarks/syntaxgym/README.md

```diff
@@ -0,0 +1,72 @@
+SyntaxGym is a unified platform for targeted syntactic evaluation of language models. It supports all steps
+of the evaluation process, from designing test suites to visualizing final results. The goal of SyntaxGym 
+is to make psycholinguistic assessment of language models more standardized, reproducible, and accessible 
+to a wide variety of researchers.  See the [SyntaxGym Website](https://syntaxgym.org/) for more details.
+
+## YAML section for Benchmark Card
```

Comment:
```suggestion
## Benchmark: SyntaxGym
```

Reply:
Suggest changing title of card to something that would work with multiple benchmarks in one plugin, e.g. `Benchmark: <benchmark name>`

Reply:
done.